### ISSUE TITLE
Logrocket loaded seems to be detected incorrectly

### ISSUE THREAD

**Issue by bernharduw** [ian/analytics#1]
Hey, just looked at the code (didn't try it), but it seems you're checking for window.Indicative instead of LogRocket here:

https://github.com/ian/analytics/blob/main/packages/logrocket/src/logrocket.ts#L34

This will only work if you have enabled both.

**Comment by ian** [comment]
Good catch.

**Comment by ian** [comment]
@bernharduw published 0.0.5

**Comment by sb55555** [comment]
Hello @ian, it seems like the bug is still present on the latest version on NPM (0.0.5).

As you can see here on line 42: 
 https://www.npmjs.com/package/analytics-plugin-logrocket/v/0.0.5?activeTab=explore.

I've been able to make it work like this for now : 
```
    {
      ...logrocketPlugin({
        appId: '',
      }),
      loaded: function () {
        return true;
      },
    },
```
